tests/combined_push/conftest.py

```diff
@@ -133,12 +133,12 @@ def starmap_ami_meta(release_params, starmap_ami_billing_config) -> Dict[str, An
         "release": release_params,
         "billing-code-config": starmap_ami_billing_config,
         "accounts": {
-            "default": "000000",
-            "us-east-1": "121212",
```

Comment:
OK that looks like a side effect of the change
https://github.com/release-engineering/pubtools-marketplacesvm/pull/26/files
We tied to use similar syntax as in pubtools-ami; going to check how they use accounts

Reply:
They used
```
--accounts {"default": {"lorem": "6795REDACTED", \
                        "ipsum": "6840REDACTED"}}
//snip                        
--snapshot-account-ids   {"default": ["6795REDACTED", "6840REDACTED"]} 
``` 

Using dict was nice how one can identify owner of numerical account id for accounts.
(one cannot have comments in JSON)

@JAVGan using list introduced in !26 as a bugfix is exactly opposite of benefit
for 1 to 1 mapping of `human readable name` to `AWS numeric account id`.

So probable !26 was not correct and we just had insufficient validation of data for `accounts` in starmap

Reply:
Yeah, probably... But should we try to revert it back to how it is used in `pubtools-ami`?

Reply:
Then I can take advantage of this PR and change it to how it's supposed to be.. We'll just need to enforce the validation in StArMap then

Reply:
Also I don't quite like their format to be different from `accounts` to `snapshot-accounts`, I would prefer either a `dict` or a `list` for both as we use a common method to parse them and will keep our mappings more consistent

Reply:
Well, anyway I went ahead and fixed that on d038c46

It now supports either format, being a `Dict[str, str]` or `Dict[str, List[str]]`.

PTAL again.

Now we will not suffer with the formats again 😅

Reply:
the case for `sharing_accounts` is obvious and unchanged  `List[str]`
and lets keep it simple for the dict `accounts`:  `Dict[str, str]`
key will he human readable name and value will be AWS account id.
e.g.
```
"accounts": {
      "marketplace02@amazon.com": "684062674729", "marketplace03@amazon.com": "200836235822",
       marketplace@amazon.com": "679593333241"}
```